lib/handlers/error-pages.js

```diff
@@ -9,12 +9,36 @@ function handler (err, req, res, next) {
 
   var ldp = req.app.locals.ldp
 
-  if (err.status === 401 &&
-      req.accepts('text/html') &&
-      ldp.auth === 'oidc') {
-    debug('401 error - redirect to Select Provider')
-    res.status(err.status)
-    redirectToLogin(req, res, next)
+  if (ldp.auth === 'oidc') {
+    let statusCode = err.status || err.statusCode || 500
```

Comment:
I think we've talked about this before, but I can't think of an instance in which, in application code, a `500` error code should be explicitly set.